src/repositories/schedule.ts

```diff
@@ -1,46 +1,87 @@
-import { doc, addDoc, getDoc, getDocs, collection } from 'firebase/firestore'
+import {
+  doc,
+  addDoc,
+  getDoc,
+  getDocs,
+  collection,
+  setDoc,
+  deleteDoc,
+} from 'firebase/firestore'
 import { db } from '../lib/firebase'
-import { Schedule } from '@/type/common'
+import { Schedule, scheduleSchema } from '@/type/common'
 
-// スケジュールを追加する関数
-export async function addSchedule(data: Schedule) {
-  try {
-    const docRef = await addDoc(collection(db, 'schedules'), data)
-    return docRef
-  } catch (error) {
-    console.error('データの追加中にエラーが発生しました:', error)
-  }
-  return null
+export const getCurrentUnixTime = (): number => {
+  const now = new Date()
+  const unixTime = Math.floor(now.getTime() / 1000)
```

Comment:
```suggestion
  const unixTime = now.getTime()
```

1000で割っちゃうと丸め誤差が怖いのでそのまま使って欲しい

Reply:
[f590d6a](https://github.com/NUTFes/chosei-chan/pull/40/commits/f590d6a738123f004e97b15d9437f8d4535860bb)